tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -41,7 +41,7 @@ func TestCreateDestroyWindowsInstance(t *testing.T) {
 	assert.NoError(t, err, "error creating Windows instance")
 
 	// Check instance and security group information in windows-node-installer.json.
-	info, err := resource.ReadInstallerInfo(dir + "windows-node-installer.json")
+	info, err := resource.ReadInstallerInfo(dir + "/" + "windows-node-installer.json")
```

Comment:
nit: I wonder how many concatenations before we should start using `join`

Reply:
just realized, shouldn't we check is "/" exist?

Reply:
Where?

Reply:
if `dir` has '/' before adding it? i know this is a quick fix, but in case Artifact_dir changes.... it might not ,,,,

Reply:
Usually environment variables won't have `/` at the end, even if we have 2 slashes in the middle, the `ReadFile` code should work fine? Have you tested in this in your unit tests?

Reply:
you are right ...

Reply:
But it will not be deleted.

Reply:
The json file will not be deleted

Reply:
i was wrong, it was fine .Thank you.

Reply:
>nit: I wonder how many concatenations before we should start using `join`

I too suggest using `join`. Is there a reason to not do so?